src/pyosmeta/parse_issues.py

```diff
@@ -265,6 +265,49 @@ def clean_gh_url(cls, user: dict[str, str]) -> dict[str, str]:
 
         return user
 
+    @field_validator(
+        "categories",
+        mode="before",
+    )
+    @classmethod
+    def clean_categories(cls, categories: list[str]) -> list[str]:
+        """Make sure each category in the list is a valid value.
+
+        Valid pyos software categories are:
+            citation-management-bibliometrics, data-retrieval,
+            data-extraction, data-processing-munging, data-deposition",
+            data-validation-testing, data-visualization-analysis,
+            workflow-automation-versioning, database-interoperability,
+            scientific-software-wrappers, geospatial, education
+
+        Parameters
+        ----------
+        categories : list[str]
+            List of categories to clean.
+
+        Returns
+        -------
+        list[str]
+            List of cleaned categories.
+        """
+
+        valid_categories = {
+            "data-processing": "data-processing-munging",
```

Comment:
Small tip: If you alphabetize these categories, they will be easier to scan. Also in the docstring.

Reply:
oh - good suggestion. i can fix that before we merge. it's a small fix and you're right more readable!

Reply:
fixed!